docs/en/advanced_guides/datasets/kitti_det.md

```diff
@@ -182,30 +183,30 @@ An example to test PointPillars on KITTI with 8 GPUs and generate a submission t
 
 - First, you need to modify the `test_evaluator` dict in your config file to add `pklfile_prefix` and `submission_prefix`, just like:
```

Comment:
First, you need to modify the `test_dataloader` and `test_evaluator` dict in your config file,  just like: